### DISCUSSION TITLE
(Feature request) Flash Cards

### DISCUSSION THREAD

**Discussion by MitchWoodin** [TriliumNext/Trilium#7336] [Plugin Ideas]
### Describe feature

A spaced repetition type of plugin would be super useful. I have enjoyed the concept in other apps but I don't see it in Trilium. The idea would be to somehow tag notes as being a flash card with a question and answer. And then the cards would randomly appear to you without revealing the answer to the question to test whether you've remembered the question or not.

**Comment by meichthys** [comment]
This is likely out of scope, but can likely be accomplished via the ETAPI and some scripting.
Perhaps a good plugin/widget idea.
